src/io/flutter/run/SdkFields.java

```diff
@@ -25,13 +25,15 @@
  */
 public class SdkFields {
   private @Nullable String filePath;
+  private @Nullable String programArgs;
   private @Nullable String workDir;
 
-  public SdkFields() {}
+  public SdkFields() {
```

Comment:
@skybrian : sorry for the unrelated formatting changes -- I ran the formatter (which we generally do) and this happened.  Hopefully not too distracting...